fish_prompt.fish

```diff
@@ -627,6 +628,16 @@ function fish_prompt -d 'bobthefish, a fish theme optimized for awesome'
   set -l __bobthefish_left_black_arrow_glyph  \uE0B2
   set -l __bobthefish_left_arrow_glyph        \uE0B3
 
+  if [ "$theme_powerline_fonts" = "no" ]
+    set __bobthefish_branch_glyph            '\u2387'
```

Comment:
let's do this without the quotes to match the rest of 'em.

otherwise, :shipit: !